src/components/PlayerForm/PlayerForm.test.jsx

```diff
@@ -0,0 +1,23 @@
+import { render, screen } from "@testing-library/react"
+import PlayerForm from "./PlayerForm.jsx"
+
+
+const testProps = {
+    edit: false,
```

Comment:
since you set a default value of `false` in your `<PlayerForm>` component, you can actually either pass nothing for it to remain false or just `edit` to be evaluated to `true`